app/lib/pre_assembly/from_staging_location/structural_builder.rb

```diff
@@ -46,7 +40,7 @@ def build
           # but if one of the files has a label, use it instead
           resource_label = fileset.label_from_file(default: default_label)
           contained_files = fileset.files.map do |fileset_file| # iterate over all the files in a resource
-            file_id = fileset_file.file_id(common_path: common_path)
+            file_id = fileset_file.relative_path
             file_attributes = {
```

Comment:
as noted above, no need for the `file_id` attribute anymore and it was removed ... bonus, we don't need to pass around `common_path` anymore either